test/jest/utils.spec.ts

```diff
@@ -218,4 +220,56 @@ describe('Utils', () => {
             expect(createCookieSyncUrl('testMPID', pixelUrl, null)).toBe('https://abc.abcdex.net/ibs:exampleid=12345&exampleuuid=testMPID&redir=');
         });
     });
+
+    describe('#inArray', () => {
+        it('returns true if the item is in the array', () => {
+            expect(inArray(['foo', 'bar', 'baz'], 'bar')).toBe(true);
+        });
+
+        it('returns false if the item is not in the array', () => {
+            expect(inArray(['foo', 'bar', 'baz'], 'qux')).toBe(false);
+        });
+
+        it('returns false if the array is empty', () => {
+            expect(inArray([], 'foo')).toBe(false);
+        });
+
+        it('returns false if the array is null', () => {
+            expect(inArray(null, 'foo')).toBe(false);
+        });
+
+        it('returns false if the array is undefined', () => {
+            expect(inArray(undefined, 'foo')).toBe(false);
+        });
+
+        it('should support different types of arrays', () => {
```

Comment:
I don't think the test naming is accurately stating what the test is doing here
```suggestion
        it('should be strict about the value type that is in the array', () => {
```

Reply:
That's not exactly true. There's no real strict typing here, but rather it accepts all types.

Reply:
hm.  the arrays here are all the same though.  what's different is the 2nd argument that you are looking for